Seminars/Sem.01/Pract.01/Task04.cpp

```diff
@@ -0,0 +1,89 @@
+// Bozhidar, Group 1, practicum date 22.02.2023
+
+#include <iostream>
+
+const int NAME_MAX_CHARS = 31;
```

Comment:
It would be better if NAME_MAX_CHARS=30 and then in  struct Student name[NAME_MAX_CHARS+1]